dll/win32/newdev/wizard.c

```diff
@@ -641,9 +641,13 @@ CHSourceDlgProc(
                 {
                     BROWSEINFO bi = { 0 };
                     LPITEMIDLIST pidl;
+                    HINSTANCE hInst = (HINSTANCE)GetWindowLong(hwndDlg, GWL_HINSTANCE);
+                    WCHAR Title[MAX_PATH];
+                    LoadStringW(hInst, IDS_BROWSE_FOR_FOLDER_TITLE, Title, MAX_PATH);
```

Comment:
Perhaps you can replace hardcoded `MAX_PATH` with `_countof` here:
```suggestion
                    LoadStringW(hInst, IDS_BROWSE_FOR_FOLDER_TITLE, Title, _countof(Title));
```

Reply:
You didn't applied this change, and probably resolved it by accident.

Reply:
OPPS. Thanks